sequencer/src/bin/espresso-dev-node.rs

```diff
@@ -438,6 +439,37 @@ mod tests {
                 .await;
         }
 
+        let large_tx = Transaction::new(100_u32.into(), vec![0; 20000]);
```

Comment:
I think to make it a proper regression test we should also send a transaction that is larger than the limit and check that it doesn't break the dev node.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1772/commits/ffecc3bad8b07357f7c4ba6c8057ce1b13007e7b